Formula/s/seam.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Seam < Formula
+  desc "This utility lets you control Seam resources"
+  homepage "https://github.com/seamapi/seam-cli"
+  url "https://github.com/seamapi/seam-cli/archive/refs/tags/0.0.33.tar.gz"
+  sha256 "5a496af613cf30dfa8a0a31bb67be7164321980dde9b4ff1575ae28deac77a56"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.local_npm_install_args
+    system "npm", "run", "build"
+    bin.install "bin/seam.cjs" => "seam"
+  end
+  test do
+    assert_match version.to_s, shell_output("#{bin}/seam --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Sure, we have a test that checks the health status, will implement that!